coverage/coverage-direct-model.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  avalon-enclave-manager:
+    volumes:
+      - ./coverage/cov-file:/project/coverage
+    command: |
+      bash -c "
+        cd /project/coverage
+        COVERAGE_FILE=.coverage.enclave-direct coverage run /usr/local/bin/enclave_manager --lmdb_url http://avalon-lmdb:9090
+        tail -f /dev/null
+      "
+
+  avalon-lmdb:
+    volumes:
+      - ./coverage/cov-file:/project/coverage
+    command: |
+      bash -c "
+        cd /project/coverage
+        COVERAGE_FILE=.coverage.lmdb-direct coverage run /usr/local/bin/kv_storage --bind http://avalon-lmdb:9090
```

Comment:
We need to avoid hard coding the port numbers in every docker-compose files. May be define the port numbers in top level docker-compose files and rest of the docker-compose files can use it.

Reply:
Let me check this.

Reply:
@manojgop I have raised github issue to handle this request (https://github.com/hyperledger/avalon/issues/515). This will be taken care seperately.